src/bioregistry/data/curated_mappings.sssom.tsv

```diff
@@ -1,115 +1,115 @@
+bioregistry:geo	Not	skos:exactMatch	obofoundry:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	ols:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	ontobee:GEO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geo	Not	skos:exactMatch	zazuko:geo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:geonames	Not	skos:exactMatch	fairsharing:FAIRsharing.56a0Uj	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gnd	Not	skos:exactMatch	prefixcommons:gnd	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gold	Not	skos:exactMatch	lov:gold	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	biocontext:GRO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	miriam:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:gro	Not	skos:exactMatch	obofoundry:gro	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:hp	Not	skos:exactMatch	aberowl:HP_O	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration		1457	2025-03-31
+bioregistry:hp	Not	skos:exactMatch	bioportal:HP_O	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration		1457	2025-03-31
+bioregistry:hp	Not	skos:exactMatch	lov:hpo	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration		1457	2025-03-31
+bioregistry:hpath	Not	skos:exactMatch	fairsharing:FAIRsharing.kj336a	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:icdo	Not	skos:exactMatch	aberowl:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:icdo	Not	skos:exactMatch	bioportal:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:icdo	Not	skos:exactMatch	ontobee:ICDO	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:kegg.reaction	Not	skos:exactMatch	wikidata:P665	orcid:0000-0001-9439-5346	semapv:MappingReview		1457	2025-03-31
+bioregistry:ma	Not	skos:exactMatch	zazuko:ma	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:mdm	Not	skos:exactMatch	aberowl:MDM	orcid:0000-0001-9439-5346	semapv:MappingReview		1457	2025-03-31
+bioregistry:mo	Not	skos:exactMatch	lov:mo	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:mod	Not	skos:exactMatch	bartoc:20333	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:mod	Not	skos:exactMatch	lov:mod	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration			2025-03-30
+bioregistry:mop	Not	skos:exactMatch	bartoc:20466	orcid:0000-0001-9439-5346	semapv:MappingReview		1457	2025-03-31
+bioregistry:mro	Not	skos:exactMatch	bioportal:MRO	orcid:0000-0001-9439-5346	semapv:ManualMappingCuration		1457	2025-03-31
```

Comment:
This was actually added in https://github.com/biopragmatics/bioregistry/pull/1262